apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.ts

```diff
@@ -17,9 +27,85 @@ router.get(
     const accessRules = await queryRows(
       sql.assessment_access_rules,
       { assessment_id: res.locals.assessment.id },
-      AssessmentAccessRulesSchema,
+      AssessmentAccessRuleRowSchema,
+    );
+
+    const assessmentPath = path.join(
+      res.locals.course.path,
+      'courseInstances',
+      res.locals.course_instance.short_name,
+      'assessments',
+      res.locals.assessment.tid,
+      'infoAssessment.json',
+    );
+
+    const assessmentPathExists = await fs.pathExists(assessmentPath);
```

Comment:
@mylesw27 under what circumstance would we be able to render this page without having a corresponding `infoAssessment.json` on disk to read from? That is, why do we need this check?